supplementary/alica_ros1/alica_tests/src/test/Expr/include/alica_tests/BlackboardTestPlan4610306063152670816.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <alica_tests/DomainCondition.h>
+#include <alica_tests/DomainPlan.h>
+#include <engine/BasicUtilityFunction.h>
+#include <engine/DefaultUtilityFunction.h>
+#include <engine/UtilityFunction.h>
+/*PROTECTED REGION ID(incl4610306063152670816) ENABLED START*/
+// Add additional includes here
+/*PROTECTED REGION END*/
+
+namespace alica
+{
+/*PROTECTED REGION ID(meth4610306063152670816) ENABLED START*/
+// Add additional options here
+/*PROTECTED REGION END*/
+class BlackboardTestPlan4610306063152670816 : public DomainPlan
+{
+public:
+    BlackboardTestPlan4610306063152670816(PlanContext& context);
+    virtual ~BlackboardTestPlan4610306063152670816();
+    /*PROTECTED REGION ID(pub4610306063152670816) ENABLED START*/
+    // Add additional public methods here
+    /*PROTECTED REGION END*/
+protected:
+    /*PROTECTED REGION ID(pro4610306063152670816) ENABLED START*/
+    // Override these methods for your use case
+    // virtual void run() override;
+    // virtual void onInit() override;
+    // virtual void onTerminate() override;
+    // Add additional protected methods here
+    /*PROTECTED REGION END*/
+private:
+    /*PROTECTED REGION ID(prv4610306063152670816) ENABLED START*/
+    // Add additional private methods here
+    /*PROTECTED REGION END*/
+};
+
+class UtilityFunction4610306063152670816 : public BasicUtilityFunction
```

Comment:
need to update the PR to catch up to devel

Reply:
Should be done after we finish getting rid of codegen relics

Reply:
I am going to open a new PR for this after we get rid of code gen in alica tests & use the new etc folder structure & folder location